website/docs/r/sks_nodepool.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: "exoscale"
+page_title: "Exoscale: exoscale_sks_nodepool"
+sidebar_current: "docs-exoscale-sks_nodepool"
+description: |-
+  Provides an Exoscale SKS Nodepool resource.
+---
+
+# exoscale\_sks\_nodepool
+
+Provides an Exoscale [SKS][sks-doc] Nodepool resource. This can be used to create, modify, and delete SKS Nodepools.
+
+
+## Example Usage
+
+```hcl
+locals {
+  zone = "ch-gva-2"
+}
+
+resource "exoscale_sks_cluster" "prod" {
+  zone    = local.zone
+  name    = "prod"
+  version = "1.20.0"
+}
+
+resource "exoscale_sks_nodepool" "ci-builders" {
```

Comment:
The instance type is missing but mandatory

Reply:
Good catch, thanks: fixed in 9014c3986048e827f00ac8b7dee638e0dc18af05.